code/game/machinery/rechargestation.dm

```diff
@@ -126,6 +126,9 @@
 	attack_hand(user)
```

Comment:
Oh no I figured it out after a while, it's line 54 in cyborg.dm but I'm leaving that in